languages/julia/exercises/concept/vehicle-purchase/.docs/introduction.md

```diff
@@ -0,0 +1,150 @@
+There are three primary conditional statements that are used in Julia:
+- `if`-statements
+- The ternary operator
+- Short-circuit evaluation
+
+## `if`-statements
+
+`if`-statements in Julia are similar to those seen in other languages:
+
+```julia
+julia> function ispositive(n)
+           if n > 0
+               println("n is positive!")
+           else
+               println("n is not positive!")
+           end
+       end
+ispositive (generic function with 1 method)
+
+julia> ispositive(10)
+n is positive!
+
+julia> ispositive(-10)
+n is not positive!
+```
```

Comment:
```suggestion
```julia
julia> function say_if_positive(n)
           if n > 0
               println("n is positive!")
           else
               println("n is not positive!")
           end
       end
say_if_positive (generic function with 1 method)

julia> say_if_positive(10)
n is positive!

julia> say_if_positive(-10)
n is not positive!
```
```